config/prod/jumpcloud-idp.yaml

```diff
@@ -0,0 +1,10 @@
+template_path: "jumpcloud-idp.yaml"
+stack_name: "jumpcloud-idp"
+stack_tags:
+  Department: "Platform"
+  Project: "Infrastructure"
+  OwnerEmail: "it@sagebase.org"
+parameters:
+  OrganizationsAdminMetadata: !file_contents idp/jumpcloud/admincentral-admin.xml
```

Comment:
The filename of the XML source should be `organizations-admin.xml` no?

Reply:
thanks, I fell asleep at the wheel.